src/lib/db/user-store.ts

```diff
@@ -113,8 +113,17 @@ class UserStore implements IUserStore {
         throw new Error('Can only find users with id, username or email.');
     }
 
+    activeAll(): any {
+        return this.db(TABLE).where({
+            deleted_at: null,
+        });
+    }
+
     activeUsers(): any {
-        return this.db(TABLE).where('deleted_at', null);
+        return this.db(TABLE).where({
+            deleted_at: null,
+            is_service: false,
+        });
```

Comment:
Reasoning for these changes is: If we're adapting `activeUsers` to only handle real users (not service accounts), then `buildSelectUser` should be adapted to still query on all active accounts (both users and SAs). Since the fields queried by `buildSelectUser` are unique this should be fine, and this also means we should be able to safely check if a username/email is unique across both users and SAs.

Reply:
Maybe we don't need to make this distinction in OSS since there should be zero SA, but I do see a problem while using the same `users` table from enterprise. You've added a new `service-account-store` in the other PR, so I'm wondering: is it worth having users and service accounts in the same table?

I'm thinking this could require additional work on validating permissions or other functionality that relies on the user's table...